components/NetworkSliceModal.tsx

```diff
@@ -58,25 +61,56 @@ const CreateNetworkSliceModal = ({ toggleModal }: NetworkSliceModalProps) => {
       .required("Selecting at least 1 gNodeB is required."),
   });
 
+  const modalTitle = (networkSliceName: string | undefined) => {
```

Comment:
```suggestion
  const modalTitle = () => {
```
Guess we don't this here, since it's already defined on a higher level, correct?